sql/wsrep_mysqld.cc

```diff
@@ -2019,7 +2019,26 @@ static int wsrep_TOI_begin(THD *thd, const char *db_, const char *table_,
   size_t buf_len(0);
   int buf_err;
 
+  DEBUG_SYNC(thd, "wsrep_TOI_begin_before");
```

Comment:
I would suggest renaming the sync points to `wsrep_TOI_begin_before_wsrep_skip_hton` and `wsrep_TOI_begin_after_wsrep_skip_hton`.

Using just `wsrep_TOI_begin_before` and `wsrep_TOI_begin_after` makes the reader think that these points are in the beginning and in the end, but they are not.

Reply:
agree, name changed